src/controllers/mainController.ts

```diff
@@ -410,6 +416,14 @@ export default class MainController implements vscode.Disposable {
     }
 
     /**
+     * Verifies the tools service version is high enough to support certain commands
+     */
+    private CanRunV2Command(): boolean {
```

Comment:
>CanRunV2Command [](start = 12, length = 15)

this method should be camelCased...we should probably change CanRunCommand at the same time